share/cmake/modules/FindOpenImageIO.cmake

```diff
@@ -59,6 +59,10 @@ find_library ( OPENIMAGEIO_LIBRARY
                NAMES OpenImageIO${OIIO_LIBNAME_SUFFIX}
                HINTS ${OPENIMAGEIO_ROOT_DIR}
                PATH_SUFFIXES lib64 lib )
+find_library ( OPENIMAGEIO_UTIL_LIBRARY
+               NAMES OpenImageIO${OIIO_LIBNAME_SUFFIX}_Util
```

Comment:
Looking at https://github.com/OpenImageIO/oiio/blob/master/src/libutil/CMakeLists.txt#L49, should this be `OpenImageIO_Util${OIIO_LIBNAME_SUFFIX}` ?

Reply:
Good catch.